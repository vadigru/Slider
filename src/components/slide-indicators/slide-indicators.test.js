import React from "react";
import renderer from "react-test-renderer";

import SlideIndicators from "./slide-indicators.jsx";

it(`Should render Arrows component`, () => {
  const indicators = [
    0,
    1,
    2,
  ];

  const tree = renderer
    .create(
        <SlideIndicators
          activeSlide={2}
          isCaption={true}
          slidesToShow={2}
          indicators={indicators}
          onIndicatorDotClick={() => {}}
        />,
        {
          createNodeMock: () => {
            return document.createElement(`div`);
          }
        }
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
