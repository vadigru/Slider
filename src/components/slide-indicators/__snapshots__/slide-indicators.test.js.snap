// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Arrows component 1`] = `
<div
  className="slider__indicators  indicators
      "
>
  <div
    className="indicators__item
            indicators__item--active"
    id={0}
    onClick={[Function]}
  >
    <div
      className="indicators__item-inner
              indicators__item-inner--active"
      onClick={[Function]}
    />
  </div>
  <div
    className="indicators__item
            "
    id={1}
    onClick={[Function]}
  >
    <div
      className="indicators__item-inner
              "
      onClick={[Function]}
    />
  </div>
  <div
    className="indicators__item
            "
    id={2}
    onClick={[Function]}
  >
    <div
      className="indicators__item-inner
              "
      onClick={[Function]}
    />
  </div>
</div>
`;
